frontend/src/metabase/query_builder/components/view/QuestionAlertWidget/QuestionAlertWidget.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/prop-types */
+import cx from "classnames";
+import { useState, useRef } from "react";
+import { t } from "ttag";
+
+import { useListCardAlertsQuery } from "metabase/api";
+import Popover from "metabase/components/Popover";
+import CS from "metabase/css/core/index.css";
+import { useSelector } from "metabase/lib/redux";
+import { canManageSubscriptions as _canManageSubscriptions } from "metabase/selectors/user";
+import { Icon } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+import type { VisualizationSettings } from "metabase-types/api";
+
+import AlertListPopoverContent from "./AlertListPopoverContent";
+import { AlertIcon } from "./QuestionAlertWidget.unit.spec";
+
+export function QuestionAlertWidget({
+  question,
+  className,
+  onCreateAlert,
+}: {
+  question: Question;
+  className?: string;
+  onCreateAlert: () => void;
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  // this isFrozen nonsense is due to AlertListPopoverContent containing a <Modal>
+  const [isFrozen, setIsFrozen] = useState(false);
+
+  const { data: questionAlerts } = useListCardAlertsQuery({
+    id: question.id(),
+  });
```

Comment:
no longer getting this from props, so it's easier to figure out where the data is coming from.